src/deluge/model/mod_controllable/mod_controllable_audio.cpp

```diff
@@ -1139,6 +1139,12 @@ int32_t ModControllableAudio::getStutterRate(ParamManager* paramManager) {
 	UnpatchedParamSet* unpatchedParams = paramManager->getUnpatchedParamSet();
 	int32_t paramValue = unpatchedParams->getValue(Param::Unpatched::STUTTER_RATE);
 
```

Comment:
the changes below adjust the quantized stutter implementation so that it does not need to change the parameter value in order to do it's calculations. When you change the parameter value and re-center it to 0, it changes what is displayed in the menu and on the mod encoder led's, which is not accurate. This caused problems when wanting to trigger stutter and adjust the value in the menu in performance view.